lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -365,6 +375,15 @@ private Map<String, Object> buildReuseResponse(AuditEventUser auditEventUser)
                         auditEventUser));
 
         LOGGER.info(LogHelper.buildLogMessage("Returning reuse journey"));
+
+        if (attainedVot.getProfileType() == OPERATIONAL_HMRC) {
+            // the only VC we should possibly have collected this session at this point is a
+            // migration VC
```

Comment:
This assumption makes me a _little_ nervous, but happy for now.

If we merge session initialisation and check existing identity, then we might be able to do this directly instead :)